scripts/v7/ch10-08-bad-script.rsc

```diff
@@ -88,10 +88,10 @@
 
     # check arg type str of correct length passed
     if (([:typeof $SiteName] != "str") or \
-        ([:len value=$SiteName] < 6)) do={
+        ([:len $SiteName] < 6)) do={
         $LogMessageFunc ("GetWebPageFunc: arg value $SiteName not a \
             valid string!");
-        :return { "duration"="*** test failed ***" };
+        :return { "*** test failed ***" };
```

Comment:
The original line 94 is correct, as an array with a named field needs to be returned for the array lookup in the later line: `:put ("Page d/load duration: " . $UrlFetch->"duration"); `